autoscale-core/src/main/java/com/hpe/caf/autoscale/core/GovernorImpl.java

```diff
@@ -178,4 +222,73 @@ private boolean shouldBeScaledDown(final int currentMemoryLoadLimit, final int s
             }
         }
     }
+
+    private static final class AdvancedInstanceInfo extends InstanceInfo
+    {
+        private int desiredInstances;
+        private double percentageDifference;
```

Comment:
Why is this stored?  Couldn't it be just calculated on demand by the `getPercentageDifference()` function?